File: AirportSim/AirportSim.Domain/Dtos/AirportDto.cs
﻿using AirportSim.Domain.Interfaces;
using System.Collections.Generic;

namespace AirportSim.Domain.Dtos
{
    public class AirportDto : IDto
    {
        public IEnumerable<AirplaneDto> Airplanes { get; set; }
        public IEnumerable<EventDto> Events { get; set; }
        public IEnumerable<StationDto> Stations { get; set; }
    }
}

File: AirportSim.Client/ClientApp/src/app/models/airport-dto.ts
import { AirplaneDto } from "./airplane-dto";
import { EventDto } from "./event-dto";
import { StationDto } from "./station-dto";

export interface AirportDto {
    airplanes:AirplaneDto[],
    events:EventDto[],
    stations:StationDto[]
}
